lib/uniara_virtual_parser/services/absences.rb

```diff
@@ -0,0 +1,28 @@
+module UniaraVirtualParser
+  module Services
+    module Absences
+      def absences(token)
+        response = Client.get_with_token('/alunos/consultas/faltas', token)
+        parse_absences response.body
+      end
+
+      private
+
+      def parse_absences(html)
+        doc = Nokogiri::HTML(html)
+        subjects = []
+
+        doc.css('div#conteudo ~table tr').each_with_index do |absence, index|
+          next if index.zero?
+          absence = Models::Absence.new(
```

Comment:
yo are using the same name here and in the parameter: `|absence, index|`
change the parameter to `absence_node` or something like this.